ChangeLog.md

```diff
@@ -27,6 +27,9 @@ See docs/process.md for more on how version tagging works.
   - Node: v10.19.0 -> v12.22.9
   - Chrome: v70 -> v74
   - Firefox: v55 -> v65
+- Minimum Firefox version was further bumped up to Firefox 68 ESR, since older
+  Firefox versions are not able to run the parallel browser harness: (#25493)
+  - Firefox: v65 -> v68
```

Comment:
Since we just did a release in the mean time can you move this up to the next one?

Reply:
check